.storybook/lucid-docs-addon/index.js

```diff
@@ -8,6 +8,11 @@ import { setOptions } from '@storybook/addon-options';
 
 const compile = marksy({ createElement: React.createElement });
 
+const isReactComponent = value =>
+	typeof value === 'function' &&
```

Comment:
is the function check necessary? I.e. will we ever hit a case where `value.prototype.isReactComponent` is true but `typeof value === 'function'` is false?